src/storage/SplitCache/InLocalStorage.js

```diff
@@ -129,7 +146,7 @@ class SplitCacheLocalStorage {
   }
 
   trafficTypeExists(trafficType) {
-    const ttCount = localStorage.getItem(this.keys.buildTrafficTypeKey(trafficType));
+    const ttCount = Number(localStorage.getItem(this.keys.buildTrafficTypeKey(trafficType)));
```

Comment:
FYI you might have used the toNumber utility we have. (Number constructor works well here so just FYI).